### PR TITLE
Migrate CDN test from E2E to integrate test suite

### PR DESCRIPTION
**Pull request by dpham692**
## What does this PR (Pull Request) do?

- [x] This PR  is not related to any Issue 

## Which Traffic Control components are affected by this PR?

- CDN in a Box
- Traffic Portal

## What is the best way to verify this PR?

https://github.com/apache/trafficcontrol/blob/master/traffic_portal/test/e2e_deprecated/CDNs/cdns-spec.js
Check the all test from link above and make sure it has in CDN test suite (traffic_portal/test/integration/data/cdn.ts)
Run the test `npm test -- --params.baseUrl https://localhost --capabilities.shardTestFiles true` and make sure all test pass.
## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [ ] This PR includes documentation OR I have explained why documentation is unnecessary
- [ ] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [ ] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_portal/test/integration/PageObjects/CDNPage.po.ts [hunk 1]
```diff
@@ -139,18 +138,18 @@ export class CDNPage extends BasePage {
...
-      if (cdn.validationMessage == value) {
-        return true;
-      } else {
-        return false;
-      }
-    })
-    return result;
+    return await basePage.GetOutputMessage().then(value => cdn.validationMessage === value);
+  }
+  public async CheckCSV(cdn: CheckCSV): Promise<boolean> {
```
Comment (ocket8888) [inline traffic_portal/test/integration/PageObjects/CDNPage.po.ts:147]: If all you need is the name, why not just take a `string`?

File: traffic_portal/test/integration/PageObjects/CDNPage.po.ts [hunk 2]
```diff
@@ -139,18 +138,18 @@ export class CDNPage extends BasePage {
...
+    let linkName = cdn.Name;
+    if (await browser.isElementPresent(element(by.xpath("//span[text()='" + linkName + "']"))) == true) {
+      return true;
+    }else{
+      return false;
+    }
```
Comment (ocket8888) [inline traffic_portal/test/integration/PageObjects/CDNPage.po.ts:153]: This `else` is unnecessary; the `if` terminates with a `return` so all execution afterward is effectively `else`.

File: traffic_portal/test/integration/specs/CDNs.spec.ts [hunk 3]
```diff
@@ -39,7 +39,12 @@ cdns.tests.forEach(async cdnsData =>{
             it('can open CDN page', async () => {
                 await cdnsPage.OpenCDNsPage();
             });
-
+             cdnsData.check.forEach(check => {
+                 it(check.description, async () => {
+                     expect(await cdnsPage.CheckCSV(check)).toBeTruthy();
```
Comment (ocket8888) [inline traffic_portal/test/integration/specs/CDNs.spec.ts:44]: `CheckCSV` is a `Promise<boolean>`, so this is not merely `Truthy`, it's `True`.

File: traffic_portal/test/integration/PageObjects/CDNPage.po.ts (hunk 1 above)
Comment (dpham692) [reply traffic_portal/test/integration/PageObjects/CDNPage.po.ts]: Fixed to string only.

File: traffic_portal/test/integration/PageObjects/CDNPage.po.ts (hunk 2 above)
Comment (dpham692) [reply traffic_portal/test/integration/PageObjects/CDNPage.po.ts]: Fix remove else.

File: traffic_portal/test/integration/specs/CDNs.spec.ts (hunk 3 above)
Comment (dpham692) [reply traffic_portal/test/integration/specs/CDNs.spec.ts]: Fix to true.

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**